lib/webhook_queue_handler.ts

```diff
@@ -180,24 +180,28 @@ export async function sendToWebhook(
             // in case of server HTTP 5xx errors we trigger a retry
             response.serverError
               ? TransientError(
-                  `Transient HTTP error calling API Proxy API: ${response.text}`
+                  `Transient HTTP error calling API Proxy: ${response.text}`
                 )
               : PermanentError(
-                  `Permanent HTTP error calling API Proxy API: ${response.text}`
+                  `Permanent HTTP error calling API Proxy: ${response.text}`
                 )
           );
         }
         return right<RuntimeError, ApiProxyResponse>(response.body);
       },
       err => {
+        const errorMsg =
+          err.response && err.response.text
+            ? err.response.text
+            : "unknown error";
         return left<RuntimeError, ApiProxyResponse>(
           err.timeout
-            ? TransientError(`Timeout calling API Proxy API`)
-            : // Fail with a permanent error in case any exception
-              // is thrown during the HTTP request
-              PermanentError(
-                `Permanent error calling API Proxy API: ${JSON.stringify(err)}`
-              )
+            ? TransientError(`Timeout calling API Proxy`)
+            : // when the server returns an HTTP 5xx error
+              err.status && Math.floor(err.status / 100) === 5
```

Comment:
to avoid the function call to `Math.floor`: `err.status % 500 < 100`

Reply:
done